File: backend/Entities/Billetera.cs
using System;
using System.Collections.Generic;

namespace Entities
{
    public partial class Billetera
    {
        public Billetera()
        {
            Transferencia = new HashSet<Transferencia>();
        }

        public int IdBilletera { get; set; }
        public int? CvuBille { get; set; }
        public int? IdUsuario { get; set; }
        public int? IdEstado { get; set; }

        public virtual Estado? IdEstadoNavigation { get; set; }
        public virtual Usuario? IdUsuarioNavigation { get; set; }
        public virtual ICollection<Transferencia> Transferencia { get; set; }
        public string NomBilletera { get;  set; }
    }
}

File: src/app/models/billetera.ts
export class Billetera {
    //     "idBilletera": 0,
    //   "cvuBille": 0,
    //   "saldoArsBille": 0,
    //   "saldoBtcBille": 0,
    //   "estadoBille": true,
    //   "idUsuario": 0,

    private idBilletera?: number;
    private cvuBilletera?: number;
    private saldoArsBille?: number;
    private saldoBtcBille?: number;
    private estadoBille: boolean = true;
    private idUsuario?: number;


    constructor(
        cvuBilletera: number,
        saldoArsBille: number,
        saldoBtcBille?: number,
        estadoBille: boolean = true,
        idUsuario?: number
    ) {
        this.cvuBilletera=cvuBilletera,
        this.saldoArsBille=saldoArsBille,
        this.saldoBtcBille=saldoBtcBille,
        this.estadoBille=estadoBille,
        this.idUsuario=idUsuario
    }
}